src/ownership.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+
+import cockpit from 'cockpit';
+
+import { FileInfo } from './fsinfo';
+
+interface Id {
+    id: number;
+    name: string | null;
+}
+
+interface IdInfo {
+    uid: Id;
+    gid: Id;
+    groups: Id[];
+}
+
+function parse_id(content: string): Id {
+    const match = content.match(/^(\d+)\(([^)]+)\)$/);
+    if (!match) {
+        throw new Error(`invalid id ${content}`);
+    }
+
+    const id = Number(match[1]);
+    const name = match[2] === "unbound" ? null : match[2];
```

Comment:
I tested running `id` as a user without a `/etc/passwd` entry and it printed "unbound" for the username.

I also noticed that `ls -l` does this now as well, instead of printing the numeric ID, which is a bit annoying.

I wonder if it's some nss module responsible for that, or a recent change in coreutils.  I tried to look into it a bit but I didn't get too far.

Reply:
Ok, `unbound` is also a fairly popular daemon so that's "interesting"

Reply:
Sounds like it might be `nss` https://github.com/search?q=repo%3Acoreutils%2Fcoreutils%20unbound&type=code

Reply:
This unbound check still feels a bit suspect to me.